R/run_prep_calculation_loans.R

```diff
@@ -287,9 +280,7 @@ run_prep_calculation_loans <- function(data_path_project_specific,
     dplyr::rename(
       production_unweighted = .data$production
     )  %>%
-    dplyr::mutate(technology_share = round(.data$technology_share, 8)) %>% # rounding errors can lead to duplicates
-    # TODO: why distinct_all?
-    dplyr::distinct_all()
+    dplyr::mutate(technology_share = round(.data$technology_share, 8))# rounding errors can lead to duplicates
```

Comment:
for my understanding, are the duplicates only on technology share? That should be unproblematic, in that case I woudl rather remove the comment, sicne it has a bit of a fix me character to it.

Reply:
I have to admit I do not fully remember where that comment came from and my guess is that it may be a wrong comment altogether. So maybe I will just remove it and also report duplicated on CUC cols